### PR TITLE
Fix some Functions docs typos.

### PR DESCRIPTION
**Pull request by etefera**

### CODE REVIEW

File: site/book/04-using-functions/01-declarative-function-execution.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ This declares two functions:
 - `kubeval` is an example validator function which you've seen before. It validates the resources
   against their OpenAPI schema.
 
-?> Refer to the [Functions Catalog][func-cat] for details on how to use a particular function.
+?> Refer to the [Functions Catalog](https://catalog.kpt.dev ':crossorigin :target=_self') for details on how to use a particular function.
```
Comment (frankfarzan) [inline site/book/04-using-functions/01-declarative-function-execution.md:40]: Doing this for each reference will be brittle as it's easy to forget crossorigin. Can you find a permanent solution? Also does it affect any external link or just the catalog?

File: site/book/04-using-functions/02-imperative-function-execution.md [hunk 2]
```diff
@@ -77,13 +77,13 @@ Note that the arguments must come after the separator `--`.
 ## Privileged Execution
 
 Since the function is provided explicitly by the user, `eval` can be more privileged and low-level
-than an declarative invocation using `render`. For example, it can optionally operate on meta
+than a declarative invocation using `render`. For example, it can optionally operate on meta
```
Comment (frankfarzan) [inline site/book/04-using-functions/02-imperative-function-execution.md:80]: I like all these changes. ty!

File: site/book/04-using-functions/01-declarative-function-execution.md (hunk 1 above)
Comment (etefera) [reply site/book/04-using-functions/01-declarative-function-execution.md:40]: Yeah, we can mark the catalog (and other common external links) as safe cross origin links at the docsify level. I'll do that here.

File: site/book/04-using-functions/02-imperative-function-execution.md (hunk 2 above)
Comment (etefera) [reply site/book/04-using-functions/02-imperative-function-execution.md:80]: Thanks!

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**